sematic/ui/src/pipelines/NotesPanel.tsx

```diff
@@ -135,7 +127,7 @@ export default function NotesPanel() {
                 <NoteView
                   note={note}
                   key={idx}
-                  author={authorsByEmailRef.current.get(note.author_id) || anonymousUser}
+                  author={note.user || anonymousUser}
```

Comment:
Perfect.
While you're at it, can you yank the `authors` list from the API endpoint response I referred to in the other comment?